auditeval/test.go

```diff
@@ -135,16 +160,18 @@ func (ts *Tests) Execute(s string) (result bool) {
 	case and, "":
 		result = true
 		for i := range res {
-			result = result && res[i]
+			result = result && res[i].TestResult
 		}
 	case or:
 		result = false
 		for i := range res {
-			result = result || res[i]
+			result = result || res[i].TestResult
 		}
 	}
+	finalOutput.TestResult = result
+	finalOutput.ActualResult = res[0].ActualResult
```

Comment:
Do we really just want the first ActualResult? Or maybe we want to concatenate them? If we genuinely only want the first one, I'm not sure there is a great deal of point in collecting them all into a list of structs, we could've just returned a list of bools plus a single string from execute().

Reply:
You're right, there is only one meaningful value here and it should be the same whichever ActualResult of res[] we pick. 

(Though clarity-wise this is another argument for not putting the results in a struct!)